lis/plugins/LIS_DAobs_pluginMod.F90

```diff
@@ -254,18 +258,14 @@ subroutine LIS_DAobs_plugin
    use SMOSNESDISsm_Mod,        only : SMOSNESDISsm_setup
 #endif
 
-#if ( defined DA_OBS_SMOS_L2SM )
```

Comment:
Why are there edits to other plugins? Maybe because you haven't pulled upstream before pushing and creating the PR? Let's consider reversing this PR tomorrow.